praw/models/reddit/subreddit.py

```diff
@@ -1545,6 +1545,50 @@ def __init__(self, subreddit):
         """Construct an instance of the Modmail object."""
         self.subreddit = subreddit
 
+    def conversations(self, after=None, limit=25, other_subreddits=None,
+                      sort=None, state=None):
+        """Return a list of conversations.
+
+        Example:
+
+        .. code:: python
+
+            conversations = reddit.subreddit('all').conversations(state='mod')
+
+        :param after: A :class:`.ModmailConversation` instance. When provided,
+            the listing begins after this conversation (default: None).
+        :param limit: The maximum number of conversations to fetch
```

Comment:
OK. I changed the documentation to note the server-side default:

>:param limit: The maximum number of conversations to fetch. If None, the server-side default is 25. (default: None).

Given that this could change, is it a good idea to document it here?